samples/fpga/rtl/rtl_multi_tasklet.py

```diff
@@ -1,11 +1,10 @@
-# Copyright 2019-2021 ETH Zurich and the DaCe authors. All rights reserved.
-"""
-    Two sequential RTL tasklets connected through a memlet.
-"""
+# Copyright 2019-2023 ETH Zurich and the DaCe authors. All rights reserved.
+#
+# Two sequential RTL tasklets connected through a memlet.
+#
+# It is intended for running simulation xilinx targets.
```

Comment:
The previous docstring version is the standard way of defining a file.

Reply:
Ah, right, I see! I've reverted them "back" to this format! :)